qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -11,6 +11,8 @@
 
 log = logging.getLogger(__name__)
 
+MDS_RESTART_GRACE = 60
```

Comment:
As the issue was the unavailibilty of metrics due to the wrong re-configuration of first clients, which is fixed now. Is there a requirement of increasing the timeout, now?

Reply:
Is it really resolved or marked resolved by mistake?

Reply:
ping?

Reply:
Your comments weren't visible earlier. If it still fails with timeout=30, there its fine to keep it 60..else no need to change it. The confusion was due to your missing comments.

Reply:
It's discussed with Venky later on and the current code is mutually agreed. Will fix the timeout later if it's seen again.